scripts/validation/pre_deployment_check.py

```
@@ -0,0 +1,739 @@
+
+    for var_name in secret_vars:
+        value = env_vars.get(var_name, "")
+
+        if not value:
+            checks.append(
+                DeploymentCheck(
+                    name=f"Secret: {var_name}",
+                    passed=False,
+                    severity="critical",
+                    message=f"{var_name} is not set",
+                    remediation=f"Generate secure value: openssl rand -hex 32",
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)